app/controllers/tag_controller.rb

```diff
@@ -110,6 +109,19 @@ def show
     end
   end
 
+
+  def show_for_author
+    if params[:id][-1..-1] == '*'
+      @wildcard = true
+    end
+    @tagname = params[:id]
+    @user = DrupalUser.find_by(name: params[:author])
```

Comment:
Can we just search `User.find_by` here, not `DrupalUser` which we're trying to reduce usage of (and eliminate)? Thanks!